prusti-specs/src/specifications/preparser.rs

```diff
@@ -131,9 +139,12 @@ impl PrustiTokenStream {
                     // to avoid later mis-identifying its suffix
                     tokens.push_back(PrustiToken::Token(token.clone()));
                     while let Some(token @ TokenTree::Punct(p)) = source.get(pos) {
+                        if p.as_char() == ',' { // Treat comma as a delimiter for macros
+                            break;
+                        }
                         pos += 1;
                         tokens.push_back(PrustiToken::Token(token.clone()));
-                        if p.spacing() != Joint {
+                        if p.spacing() != Alone {
```

Comment:
I'm not sure I understand these changes. Why stop parsing when a `Joint`-spaced token is encountered?

Reply:
Removed